### ISSUE TITLE
$$isolateBindings

### ISSUE THREAD

**Issue by anilanar** [angular-ui/ui-leaflet-draw#3]
The following line doesn't work in Angular 1.2.29:

```
// dist/ui-leaflet-draw.js, line 47
directive.$$isolateBindings[directiveName]
```

http://plnkr.co/edit/8kvQw2LZTYDa2AjAwjnc?p=info
Check dev console to see that `$$isolateBindings` doesn't exist.

**Comment by nmccready** [comment]
Yeah, I'll have to check the version number to get it to work as this is actually easier in older versions of angular. They keep switching this part of the API in 1.4-1.5 , pretty annoying.

**Comment by nmccready** [comment]
Looks like this fixes 1.4.10 as well. 1.4.10 + reverted the $isolateBindings back to allow scope yay!

**Comment by nmccready** [comment]
ty
